apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -696,7 +693,7 @@ def read(
         if value_filter is not None:
             sr.set_condition(QueryCondition(value_filter), handle.schema)
 
-        self._set_coords(sr, coords)
+        _util._set_coords(sr, coords)
```

Comment:
Maybe not on this PR (or maybe on this PR!) can we rename `sr` (which I think used to stand for _soma reader_) into something more reflective of what it currently is? Maybe as simple as `sarr` ......

Reply:
Yeah. I'll do that on this PR.